packages/server/src/utilities/rowProcessor/index.ts

```diff
@@ -349,11 +350,19 @@ export async function outputProcessing<T extends Row[] | Row>(
   }
   // remove null properties to match internal API
   const isExternal = isExternalTableID(table._id!)
-  if (isExternal) {
+  if (isExternal || db.isSqsEnabledForTenant()) {
```

Comment:
This check might cause a problem with @samwho feature flagging work - worth checking in on that.

Reply:
This has been cherry-picked from the work in `feat/pick-relationship-fields`

Reply:
Ah makes sense!